schema/json.go

```diff
@@ -74,6 +76,20 @@ func (dst *JSON) Set(src interface{}) error {
 		if err != nil {
 			return err
 		}
+
+		// For map and slice jsons, it is easier for users to work with '[]' or '{}' instead of JSON's 'null'.
+		if bytes.Equal(buf, []byte(`null`)) {
+			if isEmptyStringMap(value) {
+				*dst = JSON{Bytes: []byte("{}"), Status: Present}
+				break
```

Comment:
Does this break works? should be return imo as otherwise it will override itself with line 93? in any case lets use return.